example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -81,6 +91,26 @@ private fun onSendMetrics(otelRum: OpenTelemetryRum?) {
     counter?.add(1)
 }
 
+private fun onSendNetworkRequest(otelRum: OpenTelemetryRum?) {
+    Log.w(TAG, "making network request")
+
+    val client = OkHttpClient.Builder().build()
+    val request = Request.Builder()
+        .url("https://icanhazdadjoke.com/")
```

Comment:
it's probably better to have the test point to something we can control, right? I'm worried about this site going down, or not being accessible from a CI machine, or us accidentally flooding them with unwanted traffic or something.

Reply:
That's true. 

You and I experimented with running `python -m http.server` the other day but I ended up going with [mock-server](https://www.mock-server.com) because it was easy to set up + configure in our docker compose.